lib/widgets/fluffy_chat_app.dart

```diff
@@ -36,12 +38,19 @@ class FluffyChatApp extends StatefulWidget {
 class FluffyChatAppState extends State<FluffyChatApp> {
   bool? columnMode;
   String? _initialUrl;
+  late final networkConnectionService = getIt.get<NetworkConnectionService>();
 
   @override
-  void initState() {
+  void initState() async {
```

Comment:
In Flutter, the initState method is not allowed to be declared as async because it needs to return void and cannot be marked as async. initState is a lifecycle method in Flutter that is called when the widget is inserted into the widget tree and is typically used for initialization tasks.